cmd/find.go

```diff
@@ -74,6 +73,14 @@ func (f findMessage) JSON() string {
 func nameMatch(pattern, path string) bool {
 	matched, e := filepath.Match(pattern, filepath.Base(path))
 	errorIf(probe.NewError(e).Trace(pattern, path), "Unable to match with input pattern")
+	if !matched {
```

Comment:
When will this happen?

Reply:
It is a compatibility code.. it will happen when the filepath.Match() didn't match say for example if user said `mc find pkg --name console`